_config.yml

```diff
@@ -68,6 +68,7 @@ sass:
 # Use the following plug-ins
 gems:
   - jekyll-sitemap # Create a sitemap using the official Jekyll sitemap gem
+  - jekyll-feed # Create an Atom feed using the official Jekyll feed gem
```

Comment:
This gem is responsible now for providing feed.xml, so make sure it works with your setup.